daphne_server/src/roles/aggregator.rs

```diff
@@ -141,7 +385,7 @@ impl<S: Sync> DapAggregator<S> for crate::App {
     }
 
     fn host(&self) -> &str {
-        todo!()
+        &self.service_config.env
```

Comment:
This will usually "oxy", correct? Is that what we want, or should this be a proper hostname?

Reply:
This method is for logging purposes only, but it's also one of the "loose ends" that still needs some design work.